dali/operators/generic/erase/erase.cu

```diff
@@ -68,8 +68,11 @@ class EraseImplGpu : public OpImplBase<GPUBackend> {
   };
 
   void RunImpl(workspace_t<GPUBackend> &ws) override {
-    auto input = view<const T, Dims>(ws.template InputRef<GPUBackend>(0));
-    auto output = view<T, Dims>(ws.template OutputRef<GPUBackend>(0));
+    const auto &input_ref = ws.template InputRef<GPUBackend>(0);
+    auto &output_ref = ws.template OutputRef<GPUBackend>(0);
+    output_ref.SetLayout(input_ref.GetLayout());
```

Comment:
I'd rather use Operator's `InputLayout` function - I know that we don't have a default layout here, but it's proliferating the pattern which is not correct for the case with default input layout present.

Reply:
Hmm, the implementations are using the Operator class directly, but use the OperatorImpl as base and don't have access to this function. I will add this to the follow-up task, maybe we can do something better with it.